src/screens/NotificationsScreen.tsx

```diff
@@ -37,17 +37,16 @@ export const NotificationsScreen = () => {
   const notificationEntries = data?.notificationsForUser.edges.map(
     (edge, index) => {
       return (
-        <View key={edge.node.id}>
+        <View key={edge.node.id} style={styles.listItemView}>
           <List.Item
             title={edge.node.fullText}
             titleNumberOfLines={4}
-            style={styles.listItem}
-            // TODO: switch to i18next formatter when available
```

Comment:
Did you mean to drop this `TODO`?  I think it was for the `formatRelative` call

Reply:
I decided to remove this TODO for a couple reasons:

1. The `date-fns` function packages up quite a bit of logic/language for describing how long ago something occurred, i.e., 5 minutes ago, 2 hours ago, etc., that we do not get out of the box with intl
2. Since we haven't discussed plans to drop the `date-fns` library I no longer see the importance in swapping this out. If/when we decide to take that initiative on we will be made aware that this needs to be updated due to type errors when we uninstall the package.